binary-codec/definitions/definitions.json

```diff
@@ -1070,6 +1071,16 @@
         "type": "UInt64"
       }
     ],
+    [
```

Comment:
Are `SubjectNode` and `Subject` new entries?

Reply:
Yes I found them in xrpl.js, without them the `CredentialCreate` transaction was malformed and the XRPL rejected it.